src/migrations.js

```diff
@@ -39,5 +41,26 @@ export const migrations = [
           return warning
       }
     })
-  })
+  }),
+  async data => {
+    const ids = new Set()
+
+    function generateId () {
+      let candidate
+      do {
+        candidate = uuidV4()
+      } while (ids.has(candidate))
+
+      ids.add(candidate)
+      return candidate
+    }
```

Comment:
I have a funny feeling that this code will get repeated when things get implemented in `warnings.js`. I feel like that's going to be OK in this context because these migrations need to survive most future code changes.

Perhaps a comment explaining that is a good idea.